.github/scripts/boost/build_and_test_boost.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+BOOST_BUILD_DIR=$1
```

Comment:
Prefer to use sourcing `config.sh` so the invocation of the script requires no explicit parameters specification because the reasonable defaults will be defined there which will also ensure the placement of the source, build, test and installation artifacts to be consistent with the other packages.